backend/graphql/types/lessonType.ts

```diff
@@ -1,33 +1,49 @@
 import { gql } from "apollo-server-express";
 
-// TO DO: Define Content parameters
-
 const lessonType = gql`
-  scalar Content
+  type Content {
+    link: String
+    text: String
+  }
+
+  input ContentInput {
+    link: String
+    text: String
+  }
+
+  type ContentBlock {
+    type: String!
+    content: Content!
+  }
+
+  input ContentBlockInput {
+    type: String!
+    content: ContentInput!
+  }
```

Comment:
I'm not familiar enough with how GraphQL works to know really what's going on here. Could you explain?

Reply:
I have a small blurb on this in the "Things to Note" section of the PR description. Lmk if you have any specific questions!